test/test_proxy_tensor.py

```diff
@@ -1235,6 +1235,18 @@ def f(x, y):
         inp = (torch.randn(8)[3:], torch.randn(5))
         self.assertEqual(fx_g(*inp), f(*inp))
 
+    def test_sym_contains(self):
+        def f(x, y):
+            return x.size(0) in y
+
+        # This shouldn't raise but we need SymBool from
+        # https://github.com/pytorch/pytorch/pull/98453
+        # then modify this.  It should NOT raise a RuntimeError
+        # though!
+        self.assertRaisesRegex(NotImplementedError, "item NYI for torch.bool", lambda:
```

Comment:
nit: you can use this as a context manager instead of passing a lambda if you need to break the call anyways.